### PR TITLE
Feature: Add Admin label to participant's profile

### PR DESCRIPTION
**Pull request by KaterinaWire**
## What's new in this PR?

Add Admin label and icon to participant's profile.

### CODE REVIEW

File: Wire-iOS/Resources/Base.lproj/Localizable.strings [hunk 1]
```diff
@@ -835,6 +835,7 @@
 // User Details
 "profile.details.title" = "Details";
 "profile.details.guest" = "Guest";
+"profile.details.group_admin" = "GROUP ADMIN";
```
Comment (billypchan) [inline Wire-iOS/Resources/Base.lproj/Localizable.strings]: We can keep this in lower case and make it upper case in the code.

File: Wire-iOS/Sources/UserInterface/Components/Views/GroupRoleLabelIndicator.swift [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+public class GroupRoleLabelIndicator: UIStackView, Themeable {
```
Comment (billypchan) [inline Wire-iOS/Sources/UserInterface/Components/Views/GroupRoleLabelIndicator.swift]: Do we have snapshot for this class?

File: Wire-iOS/Resources/Base.lproj/Localizable.strings (hunk 1 above)
Comment (KaterinaWire) [reply Wire-iOS/Resources/Base.lproj/Localizable.strings]: Fixed

File: Wire-iOS/Sources/UserInterface/Components/Views/GroupRoleLabelIndicator.swift (hunk 2 above)
Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/Components/Views/GroupRoleLabelIndicator.swift]: I added a new test for this label

**Comment by billypchan** [comment]
I would recommend refactoring exist `GuestLabelIndicator` to `LabelIndicator` and init with `icon` and `title` parameter instead of creating a new `GroupRoleLabelIndicator` with similar code.

**Pull request merged**